plugins/alerts/frontend/public/javascripts/countly.models.js

```diff
@@ -177,17 +80,7 @@
         return dfd.promise();
```

Comment:
we now have a promisified ajax handler for making server request.
So you should use that to make server request instead of doing it the old way.
Here is an example - https://github.com/Countly/countly-server/blob/7303b953e9f8f1fbea391f025c8b5cf811805d7e/plugins/slipping-away-users/frontend/public/javascripts/countly.models.js#L6-L14